roles/kubernetes/master/defaults/main.yml

```diff
@@ -29,3 +29,4 @@ kube_apiserver_memory_limit: 2000M
 kube_apiserver_cpu_limit: 800m
 kube_apiserver_memory_requests: 256M
 kube_apiserver_cpu_requests: 300m
+kube_apiserver_storage_backend: etcd2
```

Comment:
Why using v2 as default ?
Kargo is using v3 since long time:
https://github.com/kubernetes-incubator/kargo/blob/master/roles/download/defaults/main.yml#L21

cc @mattymo